data/db/misc.xml

```diff
@@ -483,4 +483,21 @@
         </calibration>
     </lens>
 
+    <lens>
+        <maker>Miranda</maker>
+        <model>Miranda 28mm f/2.8 MC</model>
+        <mount>Contax/Yashica</mount>
```

Comment:
This is the list of mounts that came up when Googling this lens, but I don't know how to verify it. Is there a canonical (or at least reliable) source?